ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/**
+ `CommandBar` is a horizontal scrollable list of icon buttons divided by groups.
+ Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItem` and `trailingItem` add fixed buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
+ Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
+ */
+@objc(MSFCommandBar)
+open class CommandBar: UIView {
+    // Hierarchy:
+    //
+    // leadingButton
```

Comment:
does the view hierarchy needs to be this complex? The leading and trailing buttons can be a part of a group with single items?

|--scrollview (commandbar)
|    |--stackview (group)
|    |    |--stackview (items)

Reply:
leading and trailing buttons are fixed and can't be placed inside the scroll view.